test/e2e/data/infrastructure-vsphere-supervisor/main/topology-autoscaler/cluster-autoscaler.yaml

```diff
@@ -0,0 +1,12 @@
+- op: replace
+  path: /spec/topology/workers/machineDeployments/0/metadata
+  value:
+    annotations:
+      cluster.x-k8s.io/cluster-api-autoscaler-node-group-max-size: "5"
+      cluster.x-k8s.io/cluster-api-autoscaler-node-group-min-size: "2"
+- op: remove
+  path: /spec/topology/workers/machineDeployments/0/replicas
+  value:
+    annotations:
+      cluster.x-k8s.io/cluster-api-autoscaler-node-group-max-size: "5"
+      cluster.x-k8s.io/cluster-api-autoscaler-node-group-min-size: "2"
```

Comment:
Remove shouldn't have this value right?

Reply:
Let's keep this to align with the CAPI test. The test will set this in a later stage to 0.

Reply:
I think I'm missing something. Setting a value for a "remove" patch shouldn't make any sense? (it just drops the patch, it doesn't need a value)